app/javascript/lib/scp-api-metrics.js

```diff
@@ -84,7 +84,7 @@ export function logSearch(type, searchParams) {
     return
   }
 
-  const terms = searchParams.terms
+  const terms = searchParams.terms.split(' ')
```

Comment:
Will this work if terms are passed URL encoded? (e.g. `%20` or `+`)?

Reply:
Yup!  I just verified that "foo+bar" and "baz%20moo" are both logged as two terms.